### PR TITLE
Fix possible mistake in arrayslicefunction.md

### PR DESCRIPTION
**Pull request by ep3p**
Please, is this a mistake?

### CODE REVIEW

**Comment by YaelSchuster** [comment]
@ep3p thanks, we're checking and will get back to you shortly.

**Comment by Court72** [comment]
@shsagir @YaelSchuster 

Thanks for reviewing the proposed changes.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/arrayslicefunction.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Extracts a slice of a dynamic array.
 |--|--|--|--|
 | *array* | dynamic | &check; | THe array from which to extract the slice.|
```
Comment (YaelSchuster) [inline data-explorer/kusto/query/arrayslicefunction.md:20]:
```suggestion
| *array* | dynamic | &check; | The array from which to extract the slice.|
```

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**